src/components/Task.tsx

```diff
@@ -6,10 +6,7 @@ interface Task {
   id: string;
 }
 
-interface TaskProps {
-  title: string;
-  completed: boolean;
-  id: string;
+interface TaskProps extends Task {
```

Comment:
The nice thing about making related interfaces match is that you can then reduce repetition by extending.

Reply:
Very cool. I didn't know you could do that.